irssi/content.md

```diff
@@ -0,0 +1,40 @@
+# What is irssi?
+
+Irssi is a terminal based IRC client for UNIX systems. It also supports SILC
+and ICB protocols via plugins. Some people refer to it as 'the client of the
+future'.
+
+> [irssi.org](http://irssi.org)
+
+![logo](https://raw.githubusercontent.com/docker-library/docs/master/irssi/logo.png)
+
+# How to use this image
+
+Since no two users of irssi are likely to configure it exactly alike, this
+image does not come with any default configuration.
+
+Please refer to [upstream's excellent (and comprehensive)
+documentation](http://irssi.org/documentation) on the subject of
+configuring irssi for your needs.
+
+Be sure to also checkout the [awesome
+scripts](https://github.com/irssi/scripts.irssi.org) you can download to
+customize your irssi configuration.
+
+## Create a `Dockerfile`
+
+    FROM irssi:latest
+    COPY .irssi/ /home/user/.irssi
```

Comment:
whoops sorry forgot to use my brain when copying from haproxy, I dont think we need this even